### PR TITLE
PollSession screen updated for polling only once for a session

### PR DESCRIPTION
**Pull request by snehalPatil29**
- In Poll Session screen,
      User can now poll for a session only once
      If already polled , Overview chart of polling for that will be displayed.

### CODE REVIEW

File: app/screens/home/Questions/PollSession.js [hunk 1]
```diff
@@ -9,58 +9,107 @@ import firebase from '../../../config/firebase'
...
-        });       
+        });  
     }
 
+    checkPollResponse = () =>{
+        let thisRef = this;
+        let sessionId = this.state.sessionId;
+        let user  = this.state.feedBackGiver;
+        var query = firestoreDB.collection("feedbackResponses")
+        query = query.where("FeedBackGiver", "==" , user);
```
Comment (shaggyshelar) [inline app/screens/home/Questions/PollSession.js]: Why same variable is updated 3 times?

File: app/screens/home/Questions/PollSession.js [hunk 2]
```diff
@@ -9,58 +9,107 @@ import firebase from '../../../config/firebase'
...
+           }
+        })
+        .catch( function ( error){
+            console.log("error", error);
+        })
+    }
+    getSessionPollOverview = () => {
+        let sessionId = this.state.sessionId;
+        let thisRef = this;
+        this.state.value.map(fItem =>{
```
Comment (shaggyshelar) [inline app/screens/home/Questions/PollSession.js]: Change the variable name "value" from state. Use proper name Eg. "selectedSessionValue"

File: app/screens/home/Questions/PollSession.js [hunk 3]
```diff
@@ -9,58 +9,107 @@ import firebase from '../../../config/firebase'
...
+            if(fItem == "Yes"){
+                var query = firestoreDB.collection("feedbackResponses")
+                query = query.where("Response", "==" , fItem);
```
Comment (shaggyshelar) [inline app/screens/home/Questions/PollSession.js]: Again same variable is modified three times. Is it valid case?

File: app/screens/home/Questions/PollSession.js [hunk 4]
```diff
@@ -9,58 +9,107 @@ import firebase from '../../../config/firebase'
...
+                    })
+                })
+                .catch( function ( error){
+                    console.log("error", error);
+                })
+            }
+            else{
+                var query = firestoreDB.collection("feedbackResponses")
+                query = query.where("Response", "==" , fItem);
+                query = query.where("sessionId" ,"==" , sessionId);
```
Comment (shaggyshelar) [inline app/screens/home/Questions/PollSession.js]: Again same variable.

File: app/screens/home/Questions/PollSession.js [hunk 5]
```diff
@@ -9,58 +9,107 @@ import firebase from '../../../config/firebase'
...
-        if (this.state.Response !== "") {
-            firestoreDB.collection("Feedback_Responses")
+        if (this.state.response !== "") {
+            firestoreDB.collection("feedbackResponses")
                 .add({
-                    Question: thisRef.state.Question,
-                    Response: thisRef.state.Response,
+                    question: thisRef.state.question,
+                    Response: thisRef.state.response,
                     Date: firebase.firestore.FieldValue.serverTimestamp(),
```
Comment (shaggyshelar) [inline app/screens/home/Questions/PollSession.js]: Do not use standard keyword as property name. "DATE"

File: app/screens/home/Survey.js [hunk 6]
```diff
@@ -53,16 +53,16 @@ export class Survey extends RkComponent {
         });
     }
     onSubmitResponse = () => {
-        this.state.QueArray.forEach(fItem => {
+        this.state.queArray.forEach(fItem => {
```
Comment (shaggyshelar) [inline app/screens/home/Survey.js]: Change it to "questionsArray"

File: app/screens/home/Survey.js [hunk 7]
```diff
@@ -75,8 +75,8 @@ export class Survey extends RkComponent {
         });
     }
     onFormSelectValue = (queForm) => {
-         this.state.ShareInput = this.state.queForm.map(Fitem => {
-            this.state.QueArray.push({ Question: Fitem.QuestionTitle, Answer: new Set() });
+         this.state.renderQuestions = this.state.queForm.map(Fitem => {
```
Comment (shaggyshelar) [inline app/screens/home/Survey.js]: You should never update state variables like
this.state.variableName = VALUE.

Always modify state values using this.setState() method.

File: app/screens/home/Survey.js [hunk 8]
```diff
@@ -85,41 +85,41 @@ export class Survey extends RkComponent {
             )
         });
 
-        return this.state.ShareInput;
+        return this.state.renderQuestions;
```
Comment (shaggyshelar) [inline app/screens/home/Survey.js]: Why state variable is used here? You can use local variable also.

File: app/screens/home/profile/AttendeeProfile.js [hunk 9]
```diff
@@ -1,31 +1,223 @@
...
+import {SocialSetting} from '../../../components';
+import {FontAwesome} from '../../../assets/icons';
+import {GradientButton} from '../../../components';
+import LinkedInModal from 'react-native-linkedin';
+import firebase from '../../../config/firebase'
+
+var firestoreDB = firebase.firestore();
+
+function renderIf(condition, content) {
+    if (condition) {
```
Comment (shaggyshelar) [inline app/screens/home/profile/AttendeeProfile.js]: This files seems to be copied from profileSettings as it is. Why so?

File: app/screens/home/profile/AttendeeProfile.js [hunk 10]
```diff
@@ -1,31 +1,223 @@
...
+          'Error',
+          error.message,
+          [
+            { text: 'Ok', onPress: () => {} },
+          ],
+          { cancellable: false }
+        );
+      }
+    
+      onLinkedInConnect(token) {
```
Comment (shaggyshelar) [inline app/screens/home/profile/AttendeeProfile.js]: Is linked in functionality required here?

File: app/screens/home/Questions/PollSession.js (hunk 1 above)
Comment (snehalPatil29) [reply app/screens/home/Questions/PollSession.js]: In order to apply multiple where clauses, the same variable is updated with reference to link 
https://stackoverflow.com/questions/48036975/firestore-multiple-conditional-where-clauses?rq=1&utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

**Pull request merged**